app/presenters/saml_request_presenter.rb

```diff
@@ -0,0 +1,64 @@
+class SamlRequestPresenter
+  VALID_ATTRIBUTES = %i[
```

Comment:
I bet you could DRY and make this `ATTRIBUTE_TO_FRIENDLY_NAME_MAP.keys.freeze` if you wanted?